### PR TITLE
Changes in Download Page.

### PR DESCRIPTION
**Pull request by aashirwadRunjan**
1. Changed the choose you platform from conventional checkbox to light-looking. Looks good as it covers empty space without looking odd.
2. The box sizing in Choose your MacOS architecture was not even for AArch64 and INTEL.Fixed that issue to make it look even with windows and linux's.

### CODE REVIEW

File: package.json
Comment (Jay-Karia) [inline package.json:1]: No need to add additional dependencies and changing the version

File: src/components/download.tsx
Comment (Jay-Karia) [inline src/components/download.tsx:1]: Please fix the formatting

**Review by Jay-Karia — CHANGES_REQUESTED** [review]

File: src/components/download.tsx
Comment (aashirwadRunjan) [reply src/components/download.tsx:1]: Done

File: package.json
Comment (aashirwadRunjan) [reply package.json:1]: yes sure.Removed unwanted dependencies.

**Comment by mauro-balades** [comment]
How does it look now?

**Comment by aashirwadRunjan** [comment]
Screenshot 2024-09-03 at 1 05 37 AM
Screenshot 2024-09-03 at 1 05 43 AM

**Comment by aashirwadRunjan** [comment]
The choose your platform was also very generic and this looks cooler IMO.
The box sizing of AArch64 and x64 Intel were uneven.

**Comment by mauro-balades** [comment]
Nice, I really like it!

**Review by Jay-Karia — CHANGES_REQUESTED** [review]
`package-lock` file should also be reverted

**Comment by aashirwadRunjan** [comment]
> `package-lock` file should also be reverted

Yes sure, have removed it.

File: package-lock.json
Comment (Jay-Karia) [inline package-lock.json:1]: ?

Comment (aashirwadRunjan) [reply package-lock.json:1]: still any issue?

**Pull request merged**